.github/workflows/CD.yml

```diff
@@ -0,0 +1,83 @@
+name: CD
+
+on:
+  workflow_dispatch:
+    inputs:
+      confirmation:
+        description: 'Are you sure you want to deploy master to staging? (type yes below if so)'     
+        required: true
+        default: 'yes'
+  push:
```

Comment:
GitHub actions can run concurrently, and this runs on any push to master. Is it worthwhile using something like [turnstyle](https://github.com/softprops/turnstyle) (not sure if this is natively supported yet) to prevent running two different deployments at once?

Reply:
This makes sense. Doesn't seem to be natively supported yet so I'll put turnstyle in.